send.ts

```diff
@@ -44,7 +44,12 @@ export interface SendOptions {
   /** Try to match extensions from passed array to search for file when no
    * extension is sufficed in URL. First found is served. (defaults to
    * `undefined`) */
+
   extensions?: string[];
+  
+  /** Buffer size for sending files. There must be a good relationship between
+   * memory consumption and overhead. I recommend 32k. */
+  buffSize?: number;
```

Comment:
```suggestion
  /** The number of bytes per segment to read the file from
   * the file system.  Defaults to 32k. */
  segmentSize?: number;
```

Also try to alphabetise the properties, which now would be true.